devtools/conda-envs/rdkit-examples.yaml

```diff
@@ -26,9 +26,9 @@ dependencies:
     # Toolkit-specific
   # AmberTools 23 brings in ParmEd 4, which doesn't yet work with examples
   # https://github.com/openforcefield/openff-toolkit/issues/1532
-  - ambertools =22.4
+  - ambertools
     # https://github.com/rdkit/rdkit/issues/7221 and https://github.com/rdkit/rdkit/issues/7583
-  - rdkit !=2024.03.6,!=2024.03.5
+  - rdkit =2024
```

Comment:
IIRC these pins were because some modules (visualization?) had bad type stubs that crashed some imports. Maybe that's okay for examples?

Reply:
There are new builds out for the bad rdkit versions, so the pins shouldn't be needed. I just hadn't gotten around to allowing them in CI.